docs/UserGuide.md

```diff
@@ -98,13 +98,16 @@ Examples:
 
 Adds a transaction to the transaction book.
 
-Format: `addTxn p/PHONE_NUMBER amt/AMOUNT desc/TEST [date/DATE]`
+Format: `addTxn p/PHONE_NUMBER amt/AMOUNT desc/TEST [date/DATE] [cat/CATEGORY]...`
+
 * The `PHONE_NUMBER` refers to the phone number associated to the person had a transaction with.
-* The `AMOUNT` accepts a decimal number with up to 2 decimal places. A `-` can be added as prefix to indicate negative 
-amount.
+* The `AMOUNT` accepts a decimal number with up to 2 decimal places. A `-` can be added as prefix to indicate negative
+  amount.
 * The `DATE` accepts date formatted in the form `DDMMYYYY` i.e.`10102024`.
+* The `CATEOGORY` accepts any non-empty string.
```

Comment:
```suggestion
* The `CATEGORY` accepts non-empty strings that are alphanumeric with spaces.
```
Spelling correction and more precise definition to avoid PE UG/DG documentation bugs.